app/views/posts/index.html.erb

```diff
@@ -1,7 +1,41 @@
-<h1>All Posts</h1>
-<p><%= @placeholder_text %></p>
-<ul>
-  <% @posts.each do |post| %>
-    <li><%= link_to post.title, post_path(post) %></li>
-  <% end %>
-</ul>
+<!DOCTYPE html>
+<html>
+<head>
+  <title>All Posts</title>
+  <link rel="stylesheet" href="/assets/posts.css">
+  </head>
+<body>
+  <div class="posts-list">
+    <% @posts.each do |post| %>
+      <a href="<%= post_path(post) %>">
+        <div class="post-title">
+          <h3><%= post.title %></h3>
+          <div class="user-count">
+            <span>Comments: <%= post.comments.count %></span>
+            <span>Likes: <%= post.likes.count %></span>
+          </div>
+        </div>
+        <div class="post-details">
+          <p><%= post.content %></p>
+          <% if post.author %>
+            <p>Author: <%= post.author.name %></p>
+          <% else %>
+            <p>Author: Unknown</p>
+          <% end %>
+        </div>

```

Comment:
- [ ] When I tried to see the post#index page I got an error due to the undefined method content, please fix this issue for better UX 👇🏽 

![image](https://github.com/TayyabKM/blog-app/assets/58642949/f33712bc-34c5-4320-b3aa-5469df0430f7)

Reply:
@franclobo Can you please elaborate this and tell me URL are you trying to access?

Reply:
@franclobo I also have a question that on the all users page am I supposed to add a button or a link to navigate to the all posts page? because in the wireframes I could not see anything of that sort.

Reply:
@TayyabKM I'm sure the previous reviewer means the posts by a user,
for instance, first user:
http://127.0.0.1:3000/users/1/posts